dask/tests/test_distributed.py

```diff
@@ -549,10 +549,10 @@ async def test_shuffle_priority(c, s, a, b):
     # Make sure most "split" tasks are processing before
     # any "combine" tasks begin
     late_split = np.quantile(
-        [i for i, st in enumerate(log) if st.startswith("split")], 0.75
+        [i for i, st in enumerate(log) if st.startswith("split")], 0.73
     )
     early_combine = np.quantile(
-        [i for i, st in enumerate(log) if st.startswith("simple")], 0.25
+        [i for i, st in enumerate(log) if st.startswith("simple")], 0.27
```

Comment:
It seems like this test from #7846 has a small chance of randomly failing - This change makes the failure *slightly* less likely to occur.  Perhaps the test should just be marked as flaky, or redesigned?

Reply:
cc @jrbourbeau (in case you have thoughts)

Reply:
Yeah, this popped up over in https://github.com/dask/dask/pull/7856 (see [this CI build](https://github.com/dask/dask/pull/7856/checks?check_run_id=2974649244)). Just a guess, but could we reduce the partition / `max_branch` size such all the "split" tasks begin processing before any "combine" tasks? If so, that might give us a more deterministic thing to `assert`

Reply:
Yes - My concern is just that the test may also pass most of the time **without** the "split"-priority change.  I originally chose settings that usually fail without the modified priorities.

Reply:
Note that I opened #7879 to attack this flaky test seperately.  So, I'll roll back any changes related to the test in this PR.